docs/index.md

```diff
@@ -12,6 +12,37 @@ Use the navigation to the left to read about the available resources.
 
 ## Example Usage
 
+Terraform 0.13+ uses the Terraform Registry:
+```hcl
+terraform {
+  required_providers {
+    datadog = {
+      source  = "DataDog/datadog"
+      # version = "~> 1.0"
```

Comment:
Maybe remove that line? I'm afraid people are just going to uncomment it, and I don't really want to list a version here.

Reply:
Fair point, I originally left to show an example of where you'd put the version in this setup, but we never showed it the old way either, so removing.